Integrations/integration-Tanium.yml

```diff
@@ -0,0 +1,483 @@
+    restoreSoutAndExit(final_result)
+  type: python
+  commands:
+  - name: tn-get-package
+    arguments:
+    - name: name
+      description: Name of package
+    - name: id
+      description: Tanium id of package (to be used instead of name)
+    outputs:
+    - contextPath: Tanium.Packages.verify_expire_seconds
+      description: Timeout for verify action expiry in seconds
+    - contextPath: Tanium.Packages.display_name
+      description: Display name of package
+    - contextPath: Tanium.Packages.name
+      description: Name of created package
+    - contextPath: Tanium.Packages.command
+      description: Command to execute
+    - contextPath: Tanium.Packages.creation_time
+      description: Pacakge creation time
+    - contextPath: Tanium.Packages.command_timeout
+      description: Timeout for command execution in seconds
+    - contextPath: Tanium.Packages.modification_time
+      description: Package modification time
+    - contextPath: Tanium.Packages.last_modified_by
+      description: User who last modified packge
+    - contextPath: Tanium.Packages.last_update
+      description: Time when package was last updated
+    - contextPath: Tanium.Packages.deleted_flag
+      description: Is the package deleted
+    - contextPath: Tanium.Packages.id
+      description: Tanium unique package id
+    - contextPath: Tanium.Packages.expire_seconds
+      description: Timeout for action expiry in seconds
+    - contextPath: Tanium.Packages.files
+      description: Packge files
+    description: Get a package object based on name or ID.
+  - name: tn-get-all-packages
+    arguments: []
+    outputs:
+    - contextPath: Tanium.Packages.verify_expire_seconds
+      description: Timeout for verify action expiry in seconds
+    - contextPath: Tanium.Packages.display_name
+      description: Display name of package
+    - contextPath: Tanium.Packages.name
+      description: Name of created package
+    - contextPath: Tanium.Packages.command
+      description: Command to execute
+    - contextPath: Tanium.Packages.creation_time
+      description: Pacakge creation time
+    - contextPath: Tanium.Packages.command_timeout
+      description: Timeout for command execution in seconds
+    - contextPath: Tanium.Packages.modification_time
+      description: Package modification time
+    - contextPath: Tanium.Packages.last_modified_by
+      description: User who last modified packge
+    - contextPath: Tanium.Packages.last_update
+      description: Time when package was last updated
+    - contextPath: Tanium.Packages.deleted_flag
+      description: Is the package deleted
+    - contextPath: Tanium.Packages.id
+      description: Tanium unique package id
+    - contextPath: Tanium.Packages.expire_seconds
+      description: Timeout for action expiry in seconds
+    - contextPath: Tanium.Packages.files
+      description: Packge files
+    description: Get all Tanium package objects
+  - name: tn-get-object
+    arguments:
+    - name: name
```

Comment:
I think that one of name and id arguments should be required. and one of them should be default. No?

Reply:
you need to use either one. if I set one to required you won't be able to use it without it.